share/sv.po

```diff
@@ -0,0 +1,76 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: 1.0.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-11-16 17:49+0100\n"
+"PO-Revision-Date: 2021-11-16 17:49+0100\n"
+"Last-Translator: mats.dufberg@iis.se\n"
+"Language-Team: Zonemaster project\n"
+"Language: sv\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "Domain name required"
+msgstr "Domännamn är obligatoriskt"
+
+msgid ""
+"The domain name is not a valid IDNA string and cannot be converted to an A-"
+"label"
+msgstr ""
+"Domännamnet är inte ett giltigt IDN-namn och kan inte konverteras till A-"
+"labelformat"
+
+msgid "The domain name contains non-ascii characters and IDNA is not installed"
+msgstr ""
+"Domännamnet innehåller icke-ASCII-tecken, men stöd för IDN är inte "
+"installerat"
+
+msgid "The domain name character(s) are not supported"
+msgstr "Domännamnets teckeninnehåll stöds inte"
+
+msgid "The domain name or label is too long"
+msgstr "Domännamnet eller en domännamnsdel är för långt"
+
+msgid "Unknown profile"
+msgstr "Okänd profil"
+
+msgid "Invalid IP address"
+msgstr "Ogiltig IP-adress"
+
+msgid "Unkown language string"
+msgstr "Okänd språksträng"
+
+msgid "Language string not unique"
+msgstr "Språksträngen är inte unik"
+
+msgid "Invalid method parameter(s)."
+msgstr "Ogiltig metodparameter."
+
+msgid "Missing property"
+msgstr "Värde saknas"
+
+msgid ""
+"Warning: Zonemaster::LDNS not compiled with libidn, cannot handle non-ASCII "
+"names correctly."
+msgstr ""
+"Varning: Zonemaster::LDNS har inte kompilerats med libidn vilket gör att "
+"icke-ASCII-strängar inte kan hanteras korrekt"
+
+msgid "The domain name contains a character or characters not supported"
+msgstr "Domännamnet innehåller en eller flera tecken som inte stöds"
+
+msgid "Invalid digest format"
+msgstr "Ogiltigt format på DS-digest"
+
+msgid "Algorithm must be a positive integer"
+msgstr "Algoritm ska vara ett positivt heltal"
```

Comment:
`ska` -> `måste`

There are more like this below.

Reply:
It is also possible to use "måste" but that is not a superior alternative.

https://sv.wiktionary.org/wiki/skola#Verb

```
(modalt hjälpverb) vara tvungen att

    Man ska inte kasta skräp i naturen!
    Synonymer: måste (om satsen inte innehåller negation)
```